### PR TITLE
patch token-location

### LINKED ISSUE #219: box_fresh_auth() Error: `cache` must be a valid filename
Hi there,

Trying to use boxr to read some spreadsheets into R. When I try, I get the error; "Auto-refreshing stale OAuth token.
Error in boxGet(file_id, local_file = temp_file, version_id = version_id,  : 
  Error downloading file id xxxxxxxxx: Client error: (403) Forbidden)
  
  Apparently you have to use box_fresh_auth().
  
  I type in both  `box_fresh_auth()` and `box_fresh_auth(cache = "~/.boxr-oauth")` and I get the following error:
  Error: `cache` must be a valid filename
  
  What am I doing wrong?

### PR DESCRIPTION
**Pull request by ijlyttle**
aims to fix #219 

I still need put this into the documentation.

Basically, it looks for the token file in both locations, then if it exists only in the "bad" location, it offers to move it to the "good" location, then carries on.

@nathancday @danielruss can you let me know what you think?

Daniel - sorry to have stepped on your PR and kind offer of help - it is very much appreciated, and I look forward to hearing if this takes care of the problem.

### CODE REVIEW

File: R/boxr_auth.R [hunk 1]
```diff
@@ -629,3 +636,71 @@ get_token <- function() {
...
+        "Token file found at `{path_r_cache}`.",
+        "Move token file to new boxr standard location `{path_cache}`? (Y/n): ",
+        sep = "\n"
+      )
+      str_move <- readline(prompt = msg)
+      
+      # we are moving the file
+      if (substr(str_move, 1, 1) %in% c("Y", "y", "")) {
+        fs::file_move(path_r_cache, path_cache)
+        path <- path_cache
```
Comment (nathancday) [inline R/boxr_auth.R:690]: is `path` used later?

**Review by nathancday — APPROVED** [review]
code looks clean, thanks for smoothing this over with `harmonize`

**Comment by nathancday** [comment]
I think long term `tools::R_user_dir(..., which = 'cache')` is a more proper way to handle tokens, gonna open an issue to look at it more.

File: R/boxr_auth.R (hunk 1 above)
Comment (ijlyttle) [reply R/boxr_auth.R]: dooh! I had refactored things, but did not clean up after myself, thanks!

**Comment by ijlyttle** [comment]
I agree - I should have stressed that this is to "get us out of jail", but there's a more-thorough fix to come.

**Pull request merged**